ngrinder-core/src/main/java/net/grinder/util/NetworkUtils.java

```diff
@@ -52,9 +52,6 @@
 @SuppressWarnings("SameParameterValue")
 public abstract class NetworkUtils {
 	private static final Logger LOGGER = LoggerFactory.getLogger(NetworkUtils.class);
-	public static String DEFAULT_LOCAL_HOST_ADDRESS = getLocalHostAddress();
-	public static String DEFAULT_LOCAL_HOST_NAME = getLocalHostName();
-	public static List<InetAddress> DEFAULT_LOCAL_ADDRESSES = getAllLocalNonLoopbackAddresses(false);
```

Comment:
After trying some cases, it doesn't seem like something that should be handled on application level. I leave a comment on  https://github.com/naver/ngrinder/issues/885#issuecomment-1225240338.

After changing DNS configuration, it could derive anther [issue](https://stackoverflow.com/questions/33289695/inetaddress-getlocalhost-slow-to-run-30-seconds) that may slow down the nGrinder but, it is a kind of configuration issue also. 
Spring framework also recognizes the issue and it prints a warning log on startup like below.
```
2022-08-24 15:09:07,248 WARN  StartupInfoLogger.java:116 : InetAddress.getLocalHost().getHostName() took 5013 milliseconds to respond. Please verify your network configuration (macOS machines may need to add entries to /etc/hosts).
```

How about removing only the use of NetworkUtils constants from this PR?
Or how about just closing this PR?